### PR TITLE
Added multiple file download flag to Save-MSCatalogUpdate #12

### PR DESCRIPTION
**Pull request by JJHW149**
Feature implemented to resolve issue #12

### CODE REVIEW

File: MSCatalog/Public/Save-MSCatalogUpdate.ps1 [hunk 1]
```diff
@@ -22,6 +22,9 @@ function Save-MSCatalogUpdate {
         .PARAMETER UseBits
         If using a Windows system you can use this parameter to download the update using BITS.
 
+        .PARAMETER AcceptMultiFileUpdates
+        Allow for multiple file update downloads to occur without user interaction
```
Comment (ryan-jan) [inline MSCatalog/Public/Save-MSCatalogUpdate.ps1:26]: Can we change this to "If specified, updates containing multiple files will not prompt the user to select which file to download. Instead all files will be downloaded."

File: MSCatalog/Public/Save-MSCatalogUpdate.ps1 [hunk 2]
```diff
@@ -124,7 +130,11 @@ function Save-MSCatalogUpdate {
                 Write-Host "$Id  $FileName`r"
             }
         }
-        $SelectedId = Read-Host "Multiple files exist for this update. Enter the Id of the file to download or 'A' to download all files."
+        if($AcceptMultiFileUpdates -eq $false){
```
Comment (ryan-jan) [inline MSCatalog/Public/Save-MSCatalogUpdate.ps1:133]: Can we switch this around to be please

```
if ($AcceptMultiFileUpdates) {
    $SelectedId = "A"
} else {
    $SelectedId = Read-Host "Multiple files exist for this update. Enter the Id of the file to download or 'A' to download all files."
}
```

**Review by ryan-jan — COMMENTED** [review]
Hi Josh, thanks so much for your input on this issue. Please take a look at my comments when you get a chance. Cheers.

**Comment by JJHW149** [comment]
Thanks for the feedback Ryan, those changes have now been made.

**Pull request merged**